README.md

```diff
@@ -73,6 +73,16 @@ Follow the steps below to update your core files.
 
 ## FAQ
 
+### What’s the difference between this project and drupal/recommended-project?
+
+The two most commonly recommended Drupal project templates are this project ([drupal-composer/drupal-project](https://github.com/drupal-composer/drupal-project)) and [drupal/recommended-project](https://github.com/acquia/drupal-recommended-project).
+
+This project (drupal-composer/drupal-project) predates drupal/recommended-project and generally has more documentation, more functionality (such as Drush out of the box), and more community support.
```

Comment:
Could you please rephrase this sentence and avoid using "more"? IMO, both projects are equally good. If one is better than the other this is up to reader.

How about?

> This project (drupal-composer/drupal-project) predates drupal/recommended-project and has its own documentation, own functionality (such as Drush out of the box), and community support.

Or is this too vague/confusing?

Reply:
Per feedback from @webflo and @joachim-n below, do we just need to spell this out?

> The two most commonly recommended Drupal project templates are this project ([drupal-composer/drupal-project](https://github.com/drupal-composer/drupal-project)) and [drupal/recommended-project](https://github.com/drupal/recommended-project). Comparing the two:
>
> **drupal-composer/drupal-project**
> - Predates drupal/recommended-project
> - Documented via this README and [Github wiki](https://github.com/drupal-composer/drupal-project/wiki)
> - Supported via Github issues and pull requests
> - Includes additional features and dependencies such as Drush, Drupal Console, and an enforced minimum PHP version
>
> **drupal/recommended-project**
> - Officially recommended by the Drupal maintainers
> - Documented via [this change record](https://www.drupal.org/node/3082474)
> - Supported via drupal.org
>
> Both templates follow modern Composer and Drupal best practices (such as using Composer Scaffold) and provide a great out-of-the-box experience. See [this discussion](https://github.com/drupal-composer/drupal-project/pull/550) for additional information.

I understand that we don't want to editorialize, and I definitely don't want to step on any toes, but I'm not sure this dry comparison captures the fact that drupal-project seems way more user-friendly. The [Drupal recommended project](https://github.com/drupal/recommended-project) is positively spartan:
![screenshot-github com-2020 12 11-15_34_06](https://user-images.githubusercontent.com/1984514/101964525-8cfebc00-3bc6-11eb-8d0a-e5fcb5319807.png)
Maybe that's because discussion happens on drupal.org, but then shouldn't the repo at least mention to go there for support?

Anyway, I'm happy as long as each project at least acknowledges the other's existence so users aren't confused.

Reply:
This should probably result in a readme in the recommended-project template directory so that it shows up on the subtree splits.